server/stycoon/command_attack.go

```diff
@@ -0,0 +1,21 @@
+package stycoon
+
+import (
+	"github.com/gdg-garage/space-tycoon/server/database"
+)
+
+func NewAttackCommand(command Command) *AttackCommand {
+	return &AttackCommand{
+		Type:   command.Type,
+		Target: command.Target,
+	}
+}
+
+func (game *Game) processAttack(id int64, command Command) error {
+	c := NewAttackCommand(command)
+	err := AssertAttackCommandRequired(*c)
+	if err != nil {
+		return err
+	}
+	return database.InsertAttackCommand(game.db, id, c.Type, *c.Target)
```

Comment:
As discussed in person - we could pass the object.

counterargument: cyclic dependency. 

It is ok like this for me.